client/src/app.tsx

```diff
@@ -55,22 +83,43 @@ function App() {
   return (
     <div className="py-5" style={{ backgroundColor: "#0E0E1B" }}>
       <div className="w-fit mx-auto">
-        <HathoraLogo />
-        <div style={{ width: GameConfig.width, height: GameConfig.height }}>
+        <div className={"flex justify-center items-end"}>
+          <HathoraLogo />
+          <div className={"ml-4 text-hathoraSecondary-400 text-xl text-baseline"}>SAMPLE GAME</div>
+        </div>
+        <div className={"mt-4"} style={{ width: GameConfig.width, height: GameConfig.height }}>
           {failedToConnect ? (
-            <div className="border text-white flex flex-wrap flex-col justify-center h-full w-full content-center">
-              Failed to connect to server
+            <div className="border text-white flex flex-wrap flex-col justify-center h-full w-full content-center text-secondary-400 text-center">
+              Connection was closed
+              <br />
+              {sessionMetadata.isGameEnd ? (
+                <>
+                  <div className={"text-secondary-600"}>Game has ended</div>
+                  <div className={"text-secondary-600"}>{sessionMetadata.winningPlayer} won!</div>
```

Comment:
can we change this from "won" to "wins" lol

Reply:
this applies to old games - like if you load a game from a week ago that is over you will see this message

this is separate from when you're playing the game and it ends